### ISSUE TITLE
Show YouTube songs in search results from arbitrary URL

### ISSUE THREAD

**Issue by dbogdanov** [MeoMix/StreamusChromeExtension#205]
What about being able to query by any URL in the search form; so that the extension can parse out all youtube videos from html? I think this will be very very useful. Imagine how one can easily add videos from blog posts, facebook pages/walls, forums, whatever other web pages...  This is solving the issue #203 as well.

**Comment by jcme** [comment]
The trouble with this one is I believe we'd need to request permissions to access all sites in the manifest file in order to make a cross origin request to them. I'm not sure such a broad request for permissions would be well received.

Thoughts @MeoMix ?

**Comment by MeoMix** [comment]
Well, you can do it through an optional opt-in permission. Not all permissions have to be requested upon installation. It's the same permission request that would be needed for supporting having Streamus pause itself when audio starts coming out of another tab. That said, this issue isn't suuuper high up on my list of things to do.

**Comment by jcme** [comment]
Makes sense, thanks. An optional permission request would be a good way to go.

Slightly off topic for this thread, but it'd be great if there were a list of high priority issues that you'd like help with.I was planning on taking a look at this one as a relatively self contained feature without any UI impact (I'm not sure my CSS chops are up to making sweeping UI changes!), but I equally don't want to spend too much time on things that aren't a priority.

**Comment by MeoMix** [comment]
Totally, I'm working on it. Hopefully will have that sorted tomorrow.
